docs/getting_started/nargo/00_nargo_installation.md

```diff
@@ -145,7 +145,7 @@ There are then two approaches to proceed, differing in how the proving backend i
 
 #### Option 2.2: Install Executable with Native Backend
 
-The [barretenberg](https://github.com/AztecProtocol/barretenberg) proving backend is written in C++, hence compiling it from source would first
+The [barretenberg] proving backend is written in C++, hence compiling it from source would first
```

Comment:
@critesjosh Technically this is incorrect until we finish some work. We use a fork of aztec-connect in Noir but the work I've been doing will use the officially released version

Reply:
ok thanks for flagging. what should i update the link to?

Reply:
I've rarely seen anyone tweaking bb so I think we can leave it unless we start to get an influx of people opening issues on upstream for things that only exist in our fork.